bigbluebutton-html5/imports/ui/components/user-list/user-list-item/component.jsx

```diff
@@ -335,11 +410,15 @@ class UserListItem extends Component {
         </span>
         <span className={styles.userNameSub}>
           {userNameSub}
-          {(user.isLocked && (disablePrivateChat || disableCam || disableMic || lockedLayout || disablePublicChat)) ?
-            <span> {(user.isCurrent? " | " : null)}
+          {(user.isLocked && (disablePrivateChat
+            || disableCam
+            || disableMic
+            || lockedLayout
+            || disablePublicChat)) ?
+            <span> {(user.isCurrent ? ' | ' : null)}
               <Icon iconName='lock' />
               {intl.formatMessage(messages.locked)}
```

Comment:
The lock design is out-of-scope for this PR, but generally speaking you don't want icons without a text equivalent. If the icon is small it can be hard to tell what it actually represents and icons don't always translate to other cultures.